packages/clay-color-picker/src/index.js

```diff
@@ -0,0 +1,278 @@
+import React, {useEffect, useRef, useState} from 'react';
```

Comment:
I think we can continue to maintain the filename nomenclature, following the pattern of the other components. Something like `ClayColorPicker.js`.